feature - 10717- Phpunit test cases for ContractsParameters and ContractsWidgetVisibilityService

KavithaG-REISys opened the pull request.

sgade-reisys reviewed (changes requested):
Can you please make year values dynamic to test footer URLs so that we will not need to change the test scenario when default year/current year changes?

KavithaG-REISys commented:
> Can you please make year values dynamic to test footer URLs so that we will not need to change the test scenario when default year/current year changes?

Update the test footer url to use dynamic year parameter. Also  fixed modified  and fixed error few nycha spending and prime vendor serve test cases.
